pkg/app/app.go

```diff
@@ -89,6 +90,15 @@ func defaultFuncApiUrl(_ configuration.Configuration, logger *zerolog.Logger) co
 
 		if len(urlFromOauthToken) > 0 && len(urlFromOauthToken[0]) > 0 {
 			urlString = urlFromOauthToken[0]
+		} else if auth.IsAuthTypePAT(config.GetString(configuration.AUTHENTICATION_TOKEN)) {
+			claims, claimsErr := auth.ExtractClaimsFromPAT(config.GetString(configuration.AUTHENTICATION_TOKEN))
```

Comment:
Suggestion: to reduce the complexity of this function here, maybe already already return the url from the pat token, instead of returning the claim etc. basically encapsulate all this pat specific logic in a function. maybe even just reframe `ExtractClaimsFromPAT`